universum/configuration_support.py

```diff
@@ -8,14 +8,133 @@
     "Step",
     "Configuration",
     "Variations",
+    "combine",
```

Comment:
`__all__` refers to public API provided by module. I'd rather not add the 'combine' function as a publicly available, but also removed the legacy 'Variations' class. It would be still available for direct import, so it's not a problem; but not adding such things to `__all__` shows that we do not expect the user to use them.

Reply:
Agree with removal of Variations, but I think combine should be public - but maybe in utils.